DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests.xcodeproj/project.pbxproj

```diff
@@ -26,7 +26,8 @@
 		61E7DFB9299A5A3E001D7A3A /* Basic.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 61E7DFB8299A5A3E001D7A3A /* Basic.storyboard */; };
 		61E7DFBB299A5C9D001D7A3A /* BasicViewControllers.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61E7DFBA299A5C9D001D7A3A /* BasicViewControllers.swift */; };
 		61EC1A37299CFD7E00224FB6 /* TestUtilities in Frameworks */ = {isa = PBXBuildFile; productRef = 61EC1A36299CFD7E00224FB6 /* TestUtilities */; };
-		A77C62F429E4298200AC923B /* ImagesViewControllers.swift in Sources */ = {isa = PBXBuildFile; fileRef = A77C62F329E4298200AC923B /* ImagesViewControllers.swift */; };
```

Comment:
blocking/ When running all snapshot tests, the `testImages()` crashes:
```
Test Case '-[SRSnapshotTests.SRSnapshotTests testImages]' started.
2023-04-13 12:36:37.480228+0200 SRHost[27731:16412801] [Storyboard] Unknown class _TtC6SRHost20ImagesViewController in Interface Builder file.
2023-04-13 12:36:37.516184+0200 SRHost[27731:16412801] *** Terminating app due to uncaught exception 'NSUnknownKeyException', reason: '[<UIViewController 0x12b72f6f0> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key customButton.'
*** First throw call stack:
(
	0   CoreFoundation                      0x000000018040e7c8 __exceptionPreprocess + 172
	1   libobjc.A.dylib                     0x0000000180051144 objc_exception_throw + 56
```

It might be coming from this unnecessary change (removal of `ImagesViewControllers.swift`).

Reply:
It's been a rebase tweak I thought I resolved. Thanks for catching!